cypress/integration/my-orcid/my-orcid-side-bar.e2e.js

```diff
@@ -70,21 +73,158 @@ describe.only('My Orcid sidebar' + runInfo(), () => {
     })
   })
   describe('Websites and socials links', () => {
-    it('display a user with no items', () => {})
+    before(() => {
+      cy.cleanWebsites()
+    })
+    beforeEach(() => {
+      Cypress.Cookies.preserveOnce('XSRF-TOKEN', 'JSESSIONID')
+    })
+    after(() => {
+      cy.cleanWebsites()
+    })
+    it('display a user with no items', () => {
+      cy.get('#websites-panel').within(() => {
+        cy.get('[body=""]').should('not.exist')
+      })
+    })
     it('add items and display those with default privacy', () => {
-      // Expect changes to be display outside and inside of the modal
+      cy.get('#websites-panel')
+        .within(() => {
+          cy.get('#edit-button').click()
+        })
+        .get('#modal-container')
+        .get('#add-link')
+        .click()
+        .get('#description-input')
+        .click()
+        .type(description, { delay: 0 })
+        .get('#url-input')
+        .click()
+        .type(website, { delay: 0 })
+        .get('#save-websites-button')
+        .click()
+        .get('#websites-panel')
+        .within(() => {
+          cy.get('[body=""]')
+            .children()
+            .should('have.length', 1)
+            .get('app-panel-privacy')
+            .should(
+              'have.attr',
+              'aria-label',
+              environment.testUser.defaultPrivacy
+            )
+        })
     })
     it('remove/delete', () => {
-      // Expect changes to be display outside and inside of the modal
+      cy.get('#websites-panel')
+        .within(() => {
+          cy.get('#edit-button').click()
+        })
+        .get('#modal-container')
+        .get('#delete-button')
+        .click()
+        .get('#save-websites-button')
+        .click()
+        .wait(2000)
+        .get('#websites-panel')
+        .within(() => {
+          cy.get('[body=""]').should('not.exist')
+        })
+    })
+    it('add multiple websites', () => {
+      cy.get('#websites-panel')
+        .within(() => {
+          cy.get('#edit-button').click()
+        })
+        .get('#modal-container')
+        .get('#add-link')
+        .click()
+        .get('.mat-form-field-flex')
+        .eq(0)
+        .click({ multiple: true })
+        .type('Description Website 1', { delay: 0 })
```

Comment:
The `multiple: true ` is not require since the `.eq(0)` should return only one element

Reply:
this is not a blocking issue